src/project/qml/MuseScore/Project/ExportDialog.qml

```diff
@@ -35,32 +35,44 @@ StyledDialogView {
     title: qsTrc("project", "Export")
 
     contentWidth: 756
-    contentHeight: 386
-    margins: privateProperties.sideMargin
-
-    modal: true
+    contentHeight: 336
+    margins: 24
 
     ExportDialogModel {
         id: exportModel
     }
 
-    QtObject {
-        id: privateProperties
+    Component.onCompleted: {
+        exportModel.load()
+    }
 
-        readonly property int sideMargin: 24
+    onOpened: {
+        exportScoresListView.focusOnFirst()
     }
 
-    Component.onCompleted: {
-        exportModel.load();
+    NavigationPanel {
```

Comment:
Please move these panels to the appropriate rows, this will simplify the code

Reply:
Thanks, fixed!